apps/api/src/models/Permission.js

```diff
@@ -0,0 +1,14 @@
+import db from "./index";
+
+const mongoose = db.mongoose;
+
+const PermissionSchema = new mongoose.Schema({
+  code: String,
+  description: String,
+  type: String,
```

Comment:
You should not allowed other values than `admin` and `user` so probably change to something like:

```suggestion
  type: {
    type: String,
    enum: ["admin", "user"]
  },
```

btw I think we should always treat this string hardcoded values as constants so you can have more control in case of updating the values later

```
"user" -> PERMISSION_LEVEL.USER
"admin" -> PERMISSION_LEVEL.ADMIN
```

Reply:
you would limit the user to only have this 2 types of roles.